Essentials/src/messages.properties

```diff
@@ -73,6 +73,7 @@ confirmPayment=\u00a77To \u00a7lCONFIRM\u00a77 payment of \u00a76{0}\u00a77, ple
 connectedPlayers=\u00a76Connected players\u00a7r
 connectionFailed=Failed to open connection.
 cooldownWithMessage=\u00a74Cooldown\: {0}
+coordsAfterDeath=\u00a76Your location of death was X:{0}, Y:{1}, Z:{2}
```

Comment:
```suggestion
coordsAfterDeath=\u00a76You died in \u00a7e{0} \u00a76at \u00a7e{1}, {2}, {3}\u00a76.
```

We should provide the option to include the world name. I'm also not a fan of this presentation - specifying X: Y: Z: is redundant when almost all coordinates in Minecraft are XYZ and it's unlikely you'd mistake a Y and Z coordinate.

Reply:
I was a bit iffy on it as well but I'd agree with x, y, z more as well, like how minecraft:tp and other built-ins display it:
![image](https://user-images.githubusercontent.com/17698576/71633088-0c1dfe80-2bc7-11ea-90a2-f14d52fb6d35.png)